ublox_serialization/include/ublox/serialization.h

```diff
@@ -179,7 +179,10 @@ class Reader {
    */
   Reader(const uint8_t *data, uint32_t count, 
          const Options &options = Options()) : 
-      data_(data), count_(count), found_(false), options_(options) {}
+      data_(data), count_(count), found_(false), options_(options)
+  {
+          unused_data_.reserve(1024);
```

Comment:
Max. length is 82 bytes - but we may have multiple messages in the buffer. I believe 1kB is a reasonable value to avoid repeated resizing when pushing to the unused_data_ bufer.